ocdev/plugins/setup/setup.py

```diff
@@ -166,6 +166,12 @@ def clone_repo(self, url, branch='master', directory=None, no_history=False):
 
         if directory:
             cmd.append(directory)
-
-
-        return check_call(cmd)
\ No newline at end of file
+        assert os.getcwdu() == '/var/www', "current working directory is {0}, needs to be '/var/www'".format(os.getcwdu())
+        assert not os.path.exists('/var/www/core'), '/var/www/core already exists, cannot create test instance'
+        try:
+            result = check_call(cmd)
+        except Exception as e:
+            print(e)
+            print("likely failed due to lack of privileges")
```

Comment:
Likely is too little, you could check if the current directory is writable to be 100% sure